hammerspoon/windows.lua

```diff
@@ -1,11 +1,12 @@
 hs.window.animationDuration = 0
+window = hs.getObjectMetatable("hs.window")
```

Comment:
I've debated naming this `windowMetatable`, since that's where the functions are being defined and, you know, that's what it is. However, I don't know that that name makes the code any clearer.

If this is viable for older versions of Hammerspoon (< 0.9.79), I'm open to renaming.